src/app/shared/routing/group-route.ts

```diff
@@ -23,14 +24,27 @@ export function rawGroupRoute(id: GroupId): RawGroupRoute {
   return { contentType: 'group', id };
 }
 
+export type UserPage = 'personal-data';
+export type GroupPage = 'edit' | 'details';
+
 /**
  * Return a url array (`commands` array) to the given group, on the given page.
  */
-export function urlArrayForGroupRoute(route: RawGroupRoute, page: 'edit' | 'details' = 'details', isUser = false): UrlCommand {
+export function urlArrayForGroupRoute(
+  route: RawGroupRoute,
+  { isUser, page }: { isUser?: boolean, page?: GroupPage | UserPage } = {},
```

Comment:
The 2 parameters are not independent at all, that's a bit annoying.
Anyway, as the final goal is probably to have the page become a `string|string[]` as in items, probably no need to loose time with that.

Reply:
Fixed, I marked it as `string` for now